src/view/edit-point-view.js

```diff
@@ -1,14 +1,16 @@
-import AbstractView from '../framework/view/abstract-view.js';
+import AbstractStatefulView from '../framework/view/abstract-stateful-view.js';
 import {formatStringToDateTime, capitalize} from '../utils.js';
 import {WAYPOINT_TYPE, CITIES, POINT_EMPTY} from '../constant.js';
 
+const isOfferChecked = (offers, id) => offers.includes(id);
+
 const createOffersTemplate = ({offer, point}) => {
   const {id, title, price} = offer;
   const {offers} = point;
-  const isChecked = offers.includes(id) ? 'checked' : '';
+  const isChecked = isOfferChecked(offers, id) ? 'checked' : '';
   return (`<div class="event__offer-selector">
```

Comment:
Кажется что особого выигрыша не получил, includes стандартный метод библиотеки, не забывай еще return отбивать пустой строкой
```suggestion
  const isChecked = offers.includes(id) ? 'checked' : '';
  
  return (`<div class="event__offer-selector">
```